esti/hooks_test.go

```diff
@@ -42,6 +43,29 @@ func TestHooksSuccess(t *testing.T) {
 	})
 }
 
+func waitForListRepositoryRunsLen(ctx context.Context, t *testing.T, repo, ref string, l int) *api.ActionRunList {
```

Comment:
What action is behaving inconsistently that we need to retry it?
These retries might be hiding a real issue.

Reply:
This is a test timing issue, we query for list repository runs before the post merge record is written to the storage so that we get only the pre merge event. In a local setting with a local storage and server this race happens rarely and that's why the test in it's current state fails locally whereas in github we consistently get a single entry (for pre merge) even though we should expect 2